pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -260,10 +259,11 @@ def __init__(
             )
 
         if cpu_offload or cpu_offload_params or cpu_offload_use_pin_memory:
-            warnings.warn(
+            _warn(
```

Comment:
do we also have `_deprecate`?

Reply:
No, we haven't had a need for it. Deprecation warnings are always shown on rank 0